pt/endpoints/data/resource.py

```diff
@@ -0,0 +1,53 @@
+from datetime import date
+from flask import jsonify, request, make_response
+from flask_restful import Resource
+from utils.logging import logging
+from utils.oauth import auth, g
+from .model import Data
+from ..address.model import AMI, History
+
+
+class DatasResource(Resource):
+    # pylint: disable=R0201
+    @auth.login_required
+    def get(self):
+        if request.args.get('time'):
+            time = request.args.get('time')
+        else:
+            time = date.today()
+        logging.info(
+            "[Get Datas Request]\nUser Account:%s\nUUID:%s\n" % (g.account, g.uuid)
+        )
```

Comment:
Again, you can use f"something {some_variable}"

Reply:
This will be fix when using newer server : )